theme/js/components/search/search.vue

```diff
@@ -1,184 +1,189 @@
-            :values="facets.tag"
-            :onChange="handleSuggestorChange('tag')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-5 fr-col-lg-3">
-          <Suggestor
-            :placeholder="$t('Licenses')"
-            :searchPlaceholder="$t('Search a license...')"
-            listUrl="/datasets/licenses/"
-            :values="facets.license"
-            :onChange="handleSuggestorChange('license')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-5 fr-col-lg-2">
-          <Suggestor
-            :placeholder="$t('Formats')"
-            :searchPlaceholder="$t('Search a format...')"
-            suggestUrl="/datasets/suggest/formats/"
-            :values="facets.format"
-            :onChange="handleSuggestorChange('format')"
-          />
-        </div>
-        <div
-          class="fr-col-2 fr-col-lg-1 fr-displayed-md text-align-center text-align-right-lg"
-        >
-          <button
-            class="fr-btn fr-btn--secondary fr-btn--secondary-grey-200 text-grey-380 fr-px-2w"
-            @click="extendedForm = !extendedForm"
-            data-cy="extend-form"
-          >
-            <span v-if="!extendedForm">…</span>
-            <span v-else>X</span>
-          </button>
-        </div>
-      </div>
-      <div
-        v-if="extendedForm"
-        data-cy="extended-form"
-        class="fr-grid-row fr-grid-row--gutters fr-mt-3v align-items-center"
-      >
-        <div class="fr-col-12 fr-col-md-6 fr-col-lg-5 row-inline">
-          <Rangepicker
-            :value="facets.temporal_coverage"
-            :onChange="handleSuggestorChange('temporal_coverage')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-3">
-          <Suggestor
-            :placeholder="$t('Geographic area')"
-            :searchPlaceholder="$t('Search a geographic area...')"
-            suggestUrl="/spatial/zones/suggest/"
-            entityUrl="/spatial/zone/"
-            :values="facets.geozone"
-            :onChange="handleSuggestorChange('geozone')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-3">
-          <Suggestor
-            :placeholder="$t('Territorial granularity')"
-            :searchPlaceholder="$t('Search a granularity...')"
-            listUrl="/spatial/granularities/"
-            :values="facets.granularity"
-            :onChange="handleSuggestorChange('granularity')"
-          />
+  <form class="fr-pt-3v" @submit.prevent="search">
+    <div class="fr-grid-row fr-grid-row--middle justify-between search-bar" ref="searchRef">
+      <SearchInput
+        :onChange="handleSearchChange"
+        :value="queryString"
+        :placeholder="$t('Search for data...')"
+      />
+    </div>
+    <div class="fr-grid-row fr-mt-2w fr-pt-3v fr-mb-3v justify-between fr-grid-row--middle">
+      <p class="fr-text--lg fr-text--bold fr-my-0" v-if="totalResults">
+        {{ $t("X Results", totalResults) }}
```

Comment:
Maybe we can have a bit more spacing between total results and sort in responsive?
![image](https://user-images.githubusercontent.com/28541902/167355433-a8d4d2d5-c11d-44d6-88f7-7bfbe77a7daf.png)